src/ui/components/CreatePasscodeModule/CreatePasscodeModule.tsx

```diff
@@ -0,0 +1,211 @@
+import { BiometryErrorType } from "@aparajita/capacitor-biometric-auth";
+import { BiometryError } from "@aparajita/capacitor-biometric-auth/dist/esm/definitions";
+import { getPlatforms } from "@ionic/react";
+import { forwardRef, useEffect, useImperativeHandle, useState } from "react";
+import { Agent } from "../../../core/agent/agent";
+import { MiscRecordId } from "../../../core/agent/agent.types";
+import { BasicRecord } from "../../../core/agent/records";
+import { KeyStoreKeys, SecureStorage } from "../../../core/storage";
+import { i18n } from "../../../i18n";
+import { useAppDispatch } from "../../../store/hooks";
+import { setEnableBiometryCache } from "../../../store/reducers/biometryCache";
+import { setToastMsg } from "../../../store/reducers/stateCache";
+import { ToastMsgType } from "../../globals/types";
+import { useBiometricAuth } from "../../hooks/useBiometricsHook";
+import { Alert } from "../Alert";
+import { ErrorMessage } from "../ErrorMessage";
+import { PageFooter } from "../PageFooter";
+import { PasscodeModule } from "../PasscodeModule";
+import "./CreatePasscodeModule.scss";
+import {
+  CreatePasscodeModuleProps,
+  CreatePasscodeModuleRef,
+} from "./CreatePasscodeModule.types";
+
+const CreatePasscodeModule = forwardRef<
+  CreatePasscodeModuleRef,
+  CreatePasscodeModuleProps
+>(({ testId, title, description, onCreateSuccess, onPasscodeChange }, ref) => {
+  const dispatch = useAppDispatch();
+  const [passcode, setPasscode] = useState("");
+  const [showSetupAndroidBiometryAlert, setShowSetupAndroidBiometryAlert] =
```

Comment:
I think the way we are calling it is "Biometrics" instead of "Biometry".

Could you please check and replace all occurrences?